### ISSUE TITLE
Add support for ephemeral storage costs

### ISSUE THREAD

**Issue by jcharcalla** [kubecost/cost-analyzer-helm-chart#2213]
**What problem are you trying to solve?**

We have a request in ZD ticket: [3771](https://kubecost.zendesk.com/agent/tickets/3711) to show costs for local ephemeral storage.

Kubernetes documentation: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage

Note: In addition to local, there are CSI and Generic ephemeral volumes: https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#types-of-ephemeral-volumes

In their example they have a deployment which consumes the majority of local ephemeral storage across some number of nodes in the cluster and they would like to know the deployments cost associated with the underlying storage volume.

Cluster nodes are configured with:
```
allocatable:
  ephemeral-storage: "<Storage capacity>"
```

Deployments have the `ephemeral-storage` limit / requests:
```
resources:
  limits:
    ephemeral-storage: <Storage limit>
  requests:
    ephemeral-storage: <Storage request>
 ```

**Describe the solution you'd like**

Visibility into ephemeral storage costs on the allocations page.

With exception of Generic ephemeral volumes I believe they are similar to configMap, downwardAPI and secret volume types in that they are not associated with a PV. Because of this the solution may be a new `ephemeral-volume` allocations cost category itself  that would be visible on the allocations page and  when drilling into the Container level. The ability to enable / disable this category may be useful as well.

**Comment by kwombach12** [comment]
Thanks @jcharcalla I assume that we'd like to include this ephemeral storage in the Assets page as well?

**Comment by jcharcalla** [comment]
Yes... although this would typically reside on the root volume associated with the node, or a secondary volume mounted on the node. [See docs...](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#configurations-for-local-ephemeral-storage)

I'm not sure if that is currently tracked properly or not: https://github.com/kubecost/kubecost-cost-model/issues/628

**Comment by kwombach12** [comment]
Got it. Will have a look at this in future releases.

**Comment by jcharcalla** [comment]
Looks like there was a previous request for this: https://github.com/kubecost/cost-analyzer-helm-chart/issues/1145

**Comment by jcharcalla** [comment]
We had an additional request for this via [ZD: 4325](https://kubecost.zendesk.com/agent/tickets/4325)

**Comment by ivankube** [comment]
One more: [ZD-4807](https://kubecost.zendesk.com/agent/tickets/4807)
